### PR TITLE
X to Luisa

### PR DESCRIPTION
**Pull request by lbisinoto**
Marking my name @stalerat

### CODE REVIEW

**Comment by winpom-old** [comment]
+1

**Comment by kreynoldsmba** [comment]
+1

File: 2020_Fall/readme.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Esteves, Thais
 
-- [ ] Gontijo Bisinoto, Luisa
```
Comment (joewadcan) [inline 2020_Fall/readme.md:46]: Looks like you've missed adding the GitHub username and emoji. Just make another edit on the file, on this branch itself.

**Comment by kreynoldsmba** [comment]
my username is kreynoldsmba

**Comment by joewadcan** [comment]
Looks good thanks!

**Pull request merged**